### DISCUSSION TITLE
Missing DLL when running app on windows

### DISCUSSION THREAD

**Discussion by jordanshatford** [tauri-apps/tauri#5540] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/37837288/199844881-707b4971-6d04-4696-8aae-1e38285a8493.png)

I am seeing this error when running my tauri app on windows. Manually downloading and adding the DLL to the apps files fixes the issue, but I am curious if there is a better, more automated solution. 

I looked around in the issues and other discussions and couldnt find anything about this DLL.

Thanks for the help.

**Comment by FabianLars** [comment, marked as answer]
> I am curious if there is a better, more automated solution.

There is. Use the `msvc` rust toolchain as described in the setup doc; upgrade tauri, tauri-build, and the cli to their latest versions; make sure to clean the `src-tauri/target/` dir and build again. As long as none of your additional rust dependencies or sidecars/resources don't depend on that dll themselves it should "just work".
